src/fontra/backends/designspace.py

```diff
@@ -1139,10 +1141,14 @@ def ufoLayerToStaticGlyph(glyphSet, glyphName, penClass=PackedPathPointPen):
     pen = penClass()
     glyphSet.readGlyph(glyphName, glyph, pen, validate=False)
     components = [*pen.components] + unpackVariableComponents(glyph.lib)
+    anchors = [Anchor(name=a.get("name"), x=a["x"], y=a["y"]) for a in glyph.anchors]
```

Comment:
Ok, now I have an idea to refactor this a tiny bit better, so we can share some code with the clipboard code:

Create a new helper function called `unpackAnchors(anchors)`, call it with `glyph.anchors`, and it should produce the list of `Anchor` instances. Place the new function after `unpackVariableComponents`. The clipboard code can import it, and we've saved a tiny bit of duplication.